edgelet/iotedge/src/check/mod.rs

```diff
@@ -727,6 +728,65 @@ fn container_engine(check: &mut Check) -> Result<CheckResult, failure::Error> {
     Ok(CheckResult::Ok)
 }
 
+fn container_engine_network(check: &mut Check) -> Result<CheckResult, failure::Error> {
+    const MESSAGE: &str =
+        "Container engine is not configured for IPv6 communication.\n\
+         Please see https://docs.docker.com/v17.09/engine/userguide/networking/default_network/ipv6 \
```

Comment:
@veyalla Do we need our own docs page + aka.ms URL for this?

Reply:
I think we can redirect the docker docs via aka.ms shortened URL for brevity.

Reply:
Okay, I made https://aka.ms/iotedge-docker-ipv6

@shantanu1singh Please update the message with that.

Reply:
Done